packages/core/src/utils/anthropic-ai/index.ts

```diff
@@ -0,0 +1,242 @@
+  if (!response || typeof response !== 'object') return;
+
+  // Private response attributes that are only recorded if recordOutputs is true.
+  if (recordOutputs) {
+    // Messages.create
+    if ('content' in response) {
+      if (Array.isArray(response.content)) {
+        span.setAttributes({
+          [GEN_AI_RESPONSE_TEXT_ATTRIBUTE]: response.content
+            .map((item: { text: string | undefined }) => item.text)
+            .filter((text): text is string => text !== undefined)
+            .join(''),
+        });
+      }
+    }
+    // Completions.create
+    if ('completion' in response) {
+      span.setAttributes({ [GEN_AI_RESPONSE_TEXT_ATTRIBUTE]: response.completion });
+    }
+    // Models.countTokens
+    if ('input_tokens' in response) {
+      span.setAttributes({ [GEN_AI_RESPONSE_TEXT_ATTRIBUTE]: JSON.stringify(response.input_tokens) });
+    }
+  }
+
+  span.setAttributes({
+    [GEN_AI_RESPONSE_ID_ATTRIBUTE]: response.id,
+  });
+  span.setAttributes({
+    [GEN_AI_RESPONSE_MODEL_ATTRIBUTE]: response.model,
+  });
+  if ('created' in response && typeof response.created === 'number') {
+    span.setAttributes({
+      [ANTHROPIC_AI_RESPONSE_TIMESTAMP_ATTRIBUTE]: new Date(response.created * 1000).toISOString(),
+    });
+  }
+  if ('created_at' in response && typeof response.created_at === 'number') {
+    span.setAttributes({
+      [ANTHROPIC_AI_RESPONSE_TIMESTAMP_ATTRIBUTE]: new Date(response.created_at * 1000).toISOString(),
+    });
+  }
+
+  if (response.usage) {
+    setTokenUsageAttributes(
+      span,
+      response.usage.input_tokens,
+      response.usage.output_tokens,
+      response.usage.cache_creation_input_tokens,
+      response.usage.cache_read_input_tokens,
+    );
+  }
+}
+
+/**
+ * Get record options from the integration
+ */
+function getOptionsFromIntegration(): AnthropicAiOptions {
+  const scope = getCurrentScope();
+  const client = scope.getClient();
+  const integration = client?.getIntegrationByName(ANTHROPIC_AI_INTEGRATION_NAME) as AnthropicAiIntegration | undefined;
+  const shouldRecordInputsAndOutputs = integration ? Boolean(client?.getOptions().sendDefaultPii) : false;
+
+  return {
+    recordInputs: integration?.options?.recordInputs ?? shouldRecordInputsAndOutputs,
+    recordOutputs: integration?.options?.recordOutputs ?? shouldRecordInputsAndOutputs,
+  };
+}
+
+/**
+ * Instrument a method with Sentry spans
+ * Following Sentry AI Agents Manual Instrumentation conventions
+ * @see https://docs.sentry.io/platforms/javascript/guides/node/tracing/instrumentation/ai-agents-module/#manual-instrumentation
+ */
+function instrumentMethod<T extends unknown[], R>(
+  originalMethod: (...args: T) => Promise<R>,
+  methodPath: AnthropicAiInstrumentedMethod,
+  context: unknown,
+  options?: AnthropicAiOptions,
+): (...args: T) => Promise<R> {
+  return async function instrumentedMethod(...args: T): Promise<R> {
+    const finalOptions = options || getOptionsFromIntegration();
```

Comment:
I would rename this a bit as it's not entirely clear from the function name what this function does. It's basically just checking if the options allow the inputs/outputs to be recorded.

```suggestion
    const finalOptions = options || getRecordingOptionsFromIntegration();
```

And could it be that this is the same function? 
https://github.com/getsentry/sentry-javascript/pull/17348/files#diff-c9e6a17a3ac925c55873d82ff00c973c66d21433aec89430112dc3f10ff849cfR27

Reply:
I like the new name! And yah only one in core and one in node, not worth exporting as they are not an exact match